common/Fabric-Install-Utilities.psm1

```diff
@@ -54,7 +54,12 @@ function New-AppRoot($appDirectory, $iisUser){
         $readAccessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($iisUser, "Read", "ContainerInherit,ObjectInherit", "None", "Allow")
         $acl.AddAccessRule($writeAccessRule)
         $acl.AddAccessRule($readAccessRule)
-        Set-Acl -Path $logDirectory $acl
+        try {
+            Set-Acl -Path $logDirectory $acl
+        } catch [System.InvalidOperationException]
+        {
+           icacls $logDirectory /t /q /c /reset   
```

Comment:
Do we need to run Set-Acl again after fixing the acl with icacls?

Reply:
i don't think so.  The set updates, but leaves the ACL in a bad state (and errors).  The icacls then fixes the remaining bad state.

Reply:
Ok - cool.